app/utils/webhooks.py

```diff
@@ -361,8 +372,13 @@ def format_skipped(skipped: int) -> str:
         return f"Skipped {skipped} large attachment{'s' * (skipped != 1)}"
 
     def format(self) -> str:
-        context = (
+        original_message_info = (
+            self.author,
+            " on " * bool(self.author and self.timestamp),
```

Comment:
I think using `bool()` doesn't make it much shorter than an if expression, so maybe let's use that this time?
```suggestion
            " on " if self.author and self.timestamp else "",
```

Reply:
That is the case pretty much all the time though :P.